scripts/aggregate_cells.py

```diff
@@ -98,18 +100,22 @@ def aggregate(
     aggregate_profile_out = [
         str(out_name) for out_name in snakemake.output["aggregate_profile"]
     ]
-    meta_data_dir = str(snakemake.input["metadata"])
-    barcode = str(snakemake.input["barcodes"])
-    config_path = str(snakemake.params["aggregate_config"])
-    inputs = zip(sqlfiles, cell_count_out, aggregate_profile_out)
+    meta_data_dir = itertools.repeat(str(snakemake.input["metadata"]))
```

Comment:
i'm not familiar with this. Quick Q: will this fail if `thread=1`?

Reply:
If thread = 1, it will just use one core.